languages/csharp/exercises/concept/arrays/ArraysTests.cs

```diff
@@ -2,26 +2,60 @@
 
 public class BirdCountTests
 {
-    [Fact(Skip = "Remove this Skip property to run this test")]
+    [Fact]
```

Comment:
We should keep the Skips in the test file right? Was this a change that you made for testing purposes locally?

Reply:
Whoops. My bad. That was indeed to local testing purposes.